index.html

```diff
@@ -34,27 +34,380 @@
+                    <ul>
+                      <li>
+                        <button class="ul_li">HTML</button>
+                      </li>
+                      <li>
+                        <button class="ul_li">BOOTSTRAP</button>
+                      </li>
+                      <li>
+                        <button class="ul_li">RUBY</button>
+                      </li>
+                    </ul>
+                  </section>
+                  <footer>
+                    <button class="project"><a href="#">See Project</a></button>
+                  </footer>
+            </section>
+        </section>
+      </main>
+
+  <!-- about my self -->
+
+
+      <main class="about">
+        <section class="about_wrapper" >
+            <head>
+                <h1 class="aboutme">About me</h1>
+                <p class="aboutparagraph">
+                  Hello I’m a software developer! I can help you build a product ,
+                  feature or website Look through some of my work and experience! If you
+                  like what you see and have a project you need coded, don’t hesitate to
+                  contact me.
+                </p>
+                <button><a href="#">Get my resume</a></button>
+              </head>
+        <section class="aboutimage">
+            <section></section>
+            
+            <img src="img/aboutbackground.png" alt="">
+        </img></section>
+              <hr class="horizontal_rule"/>
+        
+             
+              <div class="container">
+                <div class="title">
+                    <div class="imagelable"><img src="./bullets/Rectangle 55.png" alt=""></div>
+                    <div><h1>Language</h1></div>
+
+                </div>
+                <div class="below">
+                    <ul>
+                        <li>Javascript</li>
+                        <li>Ruby</li>
+                        <li>HTML</li>
+                        <li>CSS</li>
+                    </ul>
+                </div>
+                <div class="title">
+                    <div class="imagelable"><img src="./bullets/Rectangle 56.png" alt=""></div>
+                    <div><h1>Frameworks</h1></div>
+
+                </div>
+                <div class="below">
+                    <ul>
+                        <li>React.js</li>
+                        <li>Ruby on rails</li>
+                        <li>RSpec</li>
+                        <li>Capybara</li>
+                    </ul>
+                </div>
+                <div class="title">
+                    <div class="imagelable"><img src="./bullets/Rectangle 57.png" alt=""></div>
+                    <div><h1>Skills</h1></div>
+
+                </div>
+                <div class="below">
+                    <ul>
+                        <li>Database Management</li>
+                        <li>Version Control</li>
+                        <li>CLI</li>
+                        <li>API Design</li>
```

Comment:
- [Optional] Kindly consider using a nested unordered list to markup your list of technologies. Note that 'languages', 'frameworks', and 'skills' are list items of the parent unordered list and 'html', 'css', and 'javascirpt' are list items under languages. Kindly use the markup below as a guide;
```html

<ul>
  <li>Languages
    <ul>
        <li>JavaScript</li>
        <li>HTML</li>
        <li>CSS</li>
    </ul>
  </li>
  <li>Frameworks
    <ul>
        <li>React</li>
        <li>Ruby on Rails</li>
        <li>Rspec</li>
    </ul>
  </li>
  <li>Skills
    <ul>
        <li>Version control</li>
    ...
    </ul>
  </li>
</ul>
```

Reply:
hi @Kamba56  thank you for this highlight